### PR TITLE
ci: Include IB frontend and stub an image

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Add deployment of Image builder to ephemeral during our PR checks

Add a stubbed Image into database directly.
For now this image is static, rebuilding it will be a followup PR.

Screenshot from 2023-07-12 12-51-23

### CODE REVIEW

File: .rhcicd/pr_check.sh [hunk 1]
```diff
@@ -36,8 +36,16 @@ CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
 curl -s $CICD_URL/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
 
 # Run ui tests
+export EXTRA_DEPLOY_ARGS="image-builder-crc"
 source "${CICD_ROOT}/deploy_ephemeral_env.sh"
 export COMPONENT_NAME="provisioning-backend"
+
+imageID="67c70300-53e1-497f-afea-656c00c812e8"
```
Comment (akhil-jha) [inline .rhcicd/pr_check.sh:43]: Maybe you need `export NAMESPACE=$(oc project -q)` after this^

File: .rhcicd/pr_check.sh [hunk 2]
```diff
@@ -36,8 +36,27 @@ CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
 curl -s $CICD_URL/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
 
 # Run ui tests
+export EXTRA_DEPLOY_ARGS="image-builder-crc"
```
Comment (mshriver) [inline .rhcicd/pr_check.sh:39]: @ezr-ondrej This comment will start a larger conversation probably appropriate for slack, but I see this being annoying to maintain weekly with QE having to either assume an image is the right one in ephemeral, or constantly keep this UUID in sync.

How about injecting this via an init container in the clowdapp template, that only executes in ephemeral?

That way its not only part of this PR check, but any ephemeral deployment of provisioning has an image available to deploy.

**Comment by akhil-jha** [comment]
GCP and AWs have been added. There are issues with azure stubbing. Merging this PR. Atleast having pr_check for GCP and AWS running is better than nothing.

**Review by akhil-jha — APPROVED** [review]

**Pull request merged**